qutip/solver/_feedback.py

```diff
@@ -0,0 +1,150 @@
+from qutip.core.cy.qobjevo import QobjEvo, _Feedback
+from qutip.core.dimensions import Dimensions, Field, SuperSpace
+import qutip.core.data as _data
+from qutip.core.qobj import Qobj
+import numpy as np
+
+
+class _ExpectFeedback(_Feedback):
+    def __init__(self, oper, default=0.):
+        self.oper = QobjEvo(oper)
+        self.N = oper.shape[1]
+        self.N2 = oper.shape[1]**2
+        self.default = default
+
+    def check_consistancy(self, dims):
```

Comment:
```suggestion
    def check_consistency(self, dims):
```
Just correcting the spelling. Also needs to be changed in a few other places.